Sources/Core/Presenter/Channel/ChannelPresenter.swift

```diff
@@ -169,7 +169,11 @@ extension ChannelPresenter {
     
     func createMessage(with text: String) -> Message {
         let messageId = editMessage?.id ?? ""
-        var attachments = uploadManager.items.compactMap({ $0.attachment })
+        
+        var attachments = uploadManager.images.isEmpty
+            ? uploadManager.files.compactMap({ $0.attachment })
+            : uploadManager.images.compactMap({ $0.attachment })
```

Comment:
Is it OK it's not possible to have a message with both images and files?

Reply:
It should be work separately.
I thought about why it's so easy to make a mistake to work with a single list of items where it's always needed two of them. Each list of items has own presentation and management.